docs/src/main/paradox/buildtools/sbt.md

```diff
@@ -25,7 +25,9 @@ you have to inherit the `.proto` definitions from `Compile` over to `Test`:
 If you have other configurations with `.proto` sources (for example `IntegrationTest`), you can enable the plugin on them:
 
 ```
-.settings(AkkaGrpcPlugin.configSettings(IntegrationTest))
+configs(IntegrationTest)
+Defaults.itSettings
+AkkaGrpcPlugin.configSettings(IntegrationTest))
```

Comment:
Good improvement. Perhaps this should be a snippet from a scripted test?

Reply:
Indeed, I was not familiar with the syntax but it looks easy - I'll amend.

Reply:
cf92528

Reply:
2bd5204